### ISSUE TITLE
How to get rid of torrentsTime on Mac

### ISSUE THREAD

**Issue by billchenxi** [torrentsTime/embed#27]
Hi there,
Can you help me to figure out how to remove this thing from Mac?

**Comment by joshlopes** [comment]
@billchenxi you have a .sh script on the the site (look at the footer for it)

**Comment by joshlopes** [comment]
actually here you have it: http://cdn.torrents-time.com/torrentsTime-uninstall.sh

**Comment by Yeehawlerz101** [comment]
Yup, I was browsing the website and its on the footer of the website.
